ui/app/components/selectable-card.js

```diff
@@ -2,16 +2,16 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 /**
  * @module SelectableCard
- * SelectableCard components are card-like components that display a title, total, subtotal, and anything after they yield.
+ * SelectableCard components are card-like components that display a title, total, subtotal, and anything after the yield.
  * They are designed to be used in containers that act as flexbox or css grid containers.
  *
  * @example
  * ```js
  * <SelectableCard @cardTitle="Tokens" @total={{totalHttpRequests}} @subText="Total" @gridContainer={{gridContainer}}/>
  * ```
- * @param cardTitle='' {String} - cardTitle displays the card title
+ * @param cardTitle=null {String} - cardTitle displays the card title
```

Comment:
Is there a reason this changed? Seems like a type of `string` should have default emptystring, not null.

Reply:
Good question.  If I didn't write in null I got a bunch of crazy characters.  Looking at other storybook examples, it looks like if you want a blank string, null is what is expected.